===================================================================
RCS file: /cvs/gstreamer/gst-plugins/ext/libpng/gstpngenc.c,v
retrieving revision 1.31.2.1
retrieving revision 1.31.2.2
diff -u -r1.31.2.1 -r1.31.2.2
--- gstpngenc.c	2005/03/10 16:57:56	1.31.2.1
+++ gstpngenc.c	2005/04/20 10:27:01	1.31.2.2
@@ -76,7 +76,7 @@
     GST_STATIC_PAD_TEMPLATE ("sink",
     GST_PAD_SINK,
     GST_PAD_ALWAYS,
-    GST_STATIC_CAPS (GST_VIDEO_CAPS_BGRA ";" GST_VIDEO_CAPS_BGR)
+    GST_STATIC_CAPS (GST_VIDEO_CAPS_RGBA ";" GST_VIDEO_CAPS_RGB)
     );
 
 static GstElementClass *parent_class = NULL;
